docs/references/kontena-yml.md

```diff
@@ -15,6 +15,124 @@ You can use environment variables in configuration values with a Bash-like ${VAR
 ## Service configuration reference
 > **Note:** Kontena supports both Docker Compose file versions respectively. Volumes and networks introduced in version 2 are not supported by Kontena yet. See more details about versioning on [Docker Compose documentation](https://docs.docker.com/compose/compose-file/#versioning)
```

Comment:
Not required. Just saying that I've assumed twice that working_dir is valid kontena.yml.

But yes, there is validation that tells you that it is not.


> On 03 Sep 2016, at 11:30, Jari Kolehmainen <notifications@github.com> wrote:
> 
> In docs/references/kontena-yml.md:
> 
> > @@ -15,6 +15,124 @@ You can use environment variables in configuration values with a Bash-like ${VAR
> >  ## Service configuration reference
> >  > **Note:** Kontena supports both Docker Compose file versions respectively. Volumes and networks introduced in version 2 are not supported by Kontena yet. See more details about versioning on [Docker Compose documentation](https://docs.docker.com/compose/compose-file/#versioning)
> Maybe we don't need that "not supported" section because we list all supported keys.. everything else is not supported?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>